ExtensionPack/README.md

```diff
@@ -3,7 +3,7 @@
 This extension pack includes a set of popular extensions for C++ development in Visual Studio Code:
 * [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)
 * [C/C++ Themes](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-themes)
-* [CMake](https://marketplace.visualstudio.com/items?itemName=twxs.cmake)
+* [CMake](https://marketplace.visualstudio.com/items?itemName=josetr.cmake-language-support-vscode)
```

Comment:
Thank you for the PR.  Can you also update the extension name in the hyperlink to `CMake Language Support`?